### DISCUSSION TITLE
Single mode question

### DISCUSSION THREAD

**Discussion by jcorporation** [MusicPlayerDaemon/MPD#1358] [Q&A] [answered]
Documentation for single state:
`Sets single state to STATE, STATE should be 0, 1 or oneshot. When single is activated, playback is stopped after current song, or song is repeated if the ‘repeat’ mode is enabled.`

Playback is not stopped with activated single oneshot mode if consume is also enabled. Behavior is:
- actual song is played
- removed from queue
- single one shot mode is disabled as expected
- next song is played (unexpected)

Is this the intended behavior and consume / single mode is exclusive or is this a bug? I tested this with current master branch.

**Comment by MaxKellermann** [comment, marked as answer]
Sounds like a bug.

**Comment by jcorporation** [comment]
Checked again - it was an error on my side
